backend/src/test/java/gov/cdc/usds/simplereport/idp/repository/DemoOktaRepositoryTest.java

```diff
@@ -313,6 +313,30 @@ void deleteOrgAndFacilities() {
         IllegalGraphqlArgumentException.class, () -> _repo.getAllUsersForOrganization(ABC));
   }
 
+  @Test
+  void reprovisionUser_success() {
+    _repo.createUser(AMOS, ABC, Set.of(ABC_1), Set.of(OrganizationRole.USER), true);
+    _repo.setUserIsActive(AMOS.getUsername(), false);
+
+    assertTrue(_repo.inactiveUsernames.contains(AMOS.getUsername()));
+
+    IdentityAttributes identityAttributes =
+        new IdentityAttributes(AMOS.getUsername(), "First", "Middle", "Last", "Jr");
+    _repo.reprovisionUser(identityAttributes);
+
+    assertFalse(_repo.inactiveUsernames.contains(AMOS.getUsername()));
+  }
+
+  @Test
+  void reprovisionUser_error() {
+    _repo.createUser(AMOS, ABC, Set.of(ABC_1), Set.of(OrganizationRole.USER), true);
+
+    IdentityAttributes identityAttributes =
+        new IdentityAttributes(AMOS.getUsername(), "First", "Middle", "Last", "Jr");
+    assertThrows(
+        IllegalGraphqlArgumentException.class, () -> _repo.reprovisionUser(identityAttributes));
```

Comment:
i'm surprised this isn't a `ConflictingUserException`

Reply:
The similar sorts of errors from the Okta layer all throw `IllegalGraphqlArgumentException`, so I followed suit.